extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/processor/SegmentGeneratorFrameProcessorFactory.java

```diff
@@ -113,6 +119,12 @@ public MSQTuningConfig getTuningConfig()
     return tuningConfig;
   }
 
+  @JsonProperty
+  public Map<String, AggregatorFactory> getDimensionToAggregatorFactoryMap()
+  {
+    return dimensionToAggregatorFactoryMap;
+  }
```

Comment:
Is this a backward-compatible change? What happens if the controller is on an older version while the workers are on a newer version?

Reply:
The field is both populated and published by the Controller. So I'm not clear where the worker's version will impact here.

Reply:
@gargvishesh , if that is true and this field need not be sent over the wire, annotate this getter as `@JsonIgnore` and don't use `@JsonProperty` annotation in the constructor for this arg either.

Reply:
A javadoc on the getter would be helpful too.